sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/models/SignedIdentifierWrapper.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.data.tables.implementation.models;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.xml.XmlReader;
+import com.azure.xml.XmlSerializable;
+import com.azure.xml.XmlToken;
+import com.azure.xml.XmlWriter;
+import java.util.ArrayList;
+import java.util.List;
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLStreamException;
+
+/**
+ * A wrapper around List&lt;SignedIdentifier&gt; which provides top-level metadata for serialization.
+ */
+public final class SignedIdentifierWrapper implements XmlSerializable<SignedIdentifierWrapper> {
```

Comment:
Should this class name be  SignedIdentifier**s**Wrapper with plural? So it overrides the deleted file.

Reply:
Given this is the name generated by Autorest and is in implementation I don't see a strong need to rename this to something better.